packages/apps/spaces/app/src/components/RootSidenav/RootSidenav.tsx

```diff
@@ -45,6 +46,7 @@ const iconMap: Record<
   Briefcase01: (props) => <Briefcase01 {...props} />,
   Building07: (props) => <Building07 {...props} />,
   CheckHeart: (props) => <CheckHeart {...props} />,
+  seed: (props) => <HeartHand {...props} />,
```

Comment:
Next time when adding table view defs pls follow the same naming convention on icon name in payload.